apps/gateway/src/middlewares/provider/index.ts

```diff
@@ -0,0 +1,28 @@
+import Elysia from "elysia";
+
+import { dbClient } from "@hebo/shared-api/middlewares/db-client";
+
+import {
+  getModelObject,
+  getProviderConfig,
+  pickChat,
+  pickEmbedding,
+} from "./service";
+
+export const provider = new Elysia({ name: "provider" })
```

Comment:
Feels like one level of redirection too much. What's the benefit of this over the same code in completions / embeddings before?